cysync/src/store/provider/syncProvider/index.tsx

```diff
@@ -590,13 +628,50 @@ export const SyncProvider: React.FC = ({ children }) => {
   const reSync = async () => {
     logger.info('Sync: ReSyncing items');
 
+    await fetchAllCustomAccounts();
     await addBalanceRefresh({});
     await addHistoryRefresh({});
     await addPriceRefresh({});
     await addLatestPriceRefresh({});
     await notifications.updateLatest();
   };
 
+  const fetchAllCustomAccounts = async () => {
+    const wallets = await walletDb.getAll();
+    for (const w of wallets) {
+      const coins = await coinDb.getAll({ walletId: w._id });
+      for (const coin of coins) {
+        const coinObj = COINS[coin.slug];
+        if (coinObj.group === CoinGroup.Near) {
+          const wallet = new NearWallet(coin.xpub, coinObj as NearCoinData);
```

Comment:
instead of fetching all coins and running the loop only for NEAR, cant we have the filter inside db call itself?